### PR TITLE
 Ban: force logout user #3696

### PR DESCRIPTION
**Pull request by xillibit**
Pull Request for Issue #3696.

### CODE REVIEW

File: libraries/kunena/user/ban.php [hunk 1]
```diff
@@ -613,7 +619,6 @@ public function save($updateOnly = false)
 			if ($block)
```
Comment (810) [inline libraries/kunena/user/ban.php]: then you can remove this check

**Comment by xillibit** [comment]
@rich20 : can-you test it please ?

**Comment by rich20** [comment]
I think, my test will not be helpful.
At ban from the page I can't see whether the user is logged out before he is banned. It's too quick.  I see logout and user is banned at the same time.

And at ban from the forum, user stay logged on.

**Comment by 810** [comment]
Yes, the user is still logged on. After banning

**Comment by 810** [comment]
With this code it worked:

```
$user = JFactory::getUser($this->userid);
$app = JFactory::getApplication ();
$app->logout((int) $this->userid);
```

**Comment by xillibit** [comment]
Updated

**Comment by 810** [comment]
thats why the user isn't logged out right now.

**Pull request merged**